mocha-multi.js

```diff
@@ -124,7 +126,9 @@ function initReportersAndStreams(runner, setup, multi) {
 
     withReplacedStdout(stream, function() {
       var Reporter = resolveReporter(reporter);
-      var r = new Reporter(shim, options || {});
+      var r = new Reporter(shim, assign({}, multi.options, {
```

Comment:
I'm not clear why multi.options is being merged here.

Why isn't it `{reporterOptions: options || {}}`?